doc/changelog/02-specification-language/12768-master+warn-non-underscore-catch-all-pattern-matching.rst

```diff
@@ -0,0 +1,7 @@
+- **Added:**
+  New preventive warning on duplicated catch-all pattern-matching
```

Comment:
I'm not in love with this wording, to me a catch-all branch is (at least) "duplicate".

IMO you should just say that we warn on unused variables in pattern matching branches.

Then, in the refman, describe as precisely as possible the current condition and convention (about `_` for example).

Reply:
I agree that this is not fully clear. I made a new attempt. See also the comment about duplication below.

I add a word in the refman. Tell me if you find it clear enough (up to the duplicate question).